python/lsst/ts/Dome/dome_csc.py

```diff
@@ -25,7 +46,8 @@
 
 
 class DomeCsc(salobj.ConfigurableCsc):
-    """Upper level Commandable SAL Component to interface with the LSST Dome lower level components.
+    """Upper level Commandable SAL Component to interface with the LSST Dome
```

Comment:
Please replace LSST with the current term, I think "Simonyi Survey Telescope"

Reply:
Good catch!